Source/Initialization/PlasmaInjector.cpp

```diff
@@ -79,14 +79,22 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
     pp.query("radially_weighted", radially_weighted);
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(radially_weighted, "ERROR: Only radially_weighted=true is supported");
 
-    // parse plasma boundaries
-    xmin = std::numeric_limits<amrex::Real>::lowest();
-    ymin = std::numeric_limits<amrex::Real>::lowest();
-    zmin = std::numeric_limits<amrex::Real>::lowest();
-
-    xmax = std::numeric_limits<amrex::Real>::max();
-    ymax = std::numeric_limits<amrex::Real>::max();
-    zmax = std::numeric_limits<amrex::Real>::max();
+    // box boundaries
+    // FIXME: Only injects particles on coarsest grid, so changes are needed to allow injection to finer grid levels.
```

Comment:
the fixme looks confusing to me at this point, where we define the geometry. It's more of a general comment for the whole plasma injector class I think.
In the annotated block, we just define the xmin/xmax et al., which are always the "widest" grid.

Reply:
Thanks Axel, that is a good point. How about?
```suggestion
    // NOTE: Default injection range is set to the dimensions of the mother grid.
```